### ISSUE TITLE
line separator is coming for page background when using header/footer html

### ISSUE THREAD

**Issue by kavishatalsania** [wkhtmltopdf/wkhtmltopdf#4666]
I need background image for whole page. I am generating PDF with `--header-html` and `--footer-html`. So I've added background image in header, middle content and footer. It renders correctly but getting line separator between header and middle content and also in middle content and footer.

![image](https://user-images.githubusercontent.com/32358748/79426479-d6b7e800-7fe0-11ea-8cdd-2dd610db677f.png)

![image](https://user-images.githubusercontent.com/32358748/79426549-eafbe500-7fe0-11ea-88a0-5eadda0521c2.png)

Thank you for your help.

**Comment by TheRaviraj** [comment]
wkhtmltopdf --footer-html /tmp/footer.html /tmp/content.html /tmp/report.pdf
The switch --footer-html, is not support using unpatched qt, and will be ignored.Loading page (1/2)
Segmentation fault====>                                      ] 36%
bash-4.4# wkhtmltopdf --footer-page /tmp/footer.html /tmp/content.html /tmp/report.pdf
Unknown long argument --footer-page

version is 
  wkhtmltopdf 0.12.5
How to install 0.12.5 (qt unpatched) at apline 3.8

**Comment by PhilterPaper** [comment]
Is this the same problem as you reported in #4665?

**Comment by kavishatalsania** [comment]
@PhilterPaper No. It is different issue. This is coming for page background image. Not able to solve with `--footer-spacing -1` hack.
